### PR TITLE
Shiva's Panic Room Nightmare Insert

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
# About the pull request

On the day of the xenomorph incursion on the colony, the alarms started blaring calling all colonists to take shelter and hide, top Administrators made a break for the Spaceport shuttles. All but one, the Assistant Operations Manager knew the shuttles would be gone by now, so they opened the panic room for all civilians who wanted to be safe from the aliens. 

Now we are in the present day, the supplies are running low, the weapons are lackluster to deal with the xenos and outside contact has been cut off, same as the power in some areas. A group of doctors, scientists and normal civilians lead by the Assistant Manager must find a way to survive before it is too late for their rescues arrival.

# Explain why it's good for the game

More nightmare inserts are great. I like the story they provide and this time I did not majorly buff them, hopefully this can be merged same as the LV one.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image

</details>

# Changelog
:cl:
mapadd: added a new nightmare insert in the Operations Panic Room on Shivas

### CODE REVIEW

File: code/modules/gear_presets/survivors/shivas_snowball/panic_room_insert_shivas.dm [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		ACCESS_WY_RESEARCH,
+		ACCESS_WY_ARMORY,
+		ACCESS_CIVILIAN_PUBLIC,
+		ACCESS_CIVILIAN_RESEARCH,
+		ACCESS_CIVILIAN_ENGINEERING,
+		ACCESS_CIVILIAN_LOGISTICS,
+		ACCESS_CIVILIAN_BRIG,
+		ACCESS_CIVILIAN_MEDBAY,
+		ACCESS_CIVILIAN_COMMAND,
+		ACCESS_ILLEGAL_PIRATE,
```
Comment (Nanu308) [inline code/modules/gear_presets/survivors/shivas_snowball/panic_room_insert_shivas.dm:26]: pirate access aka antag access?

File: maps/map_files/Ice_Colony_v3/standalone/panic_room_hold.dmm
Comment (Nanu308) [inline maps/map_files/Ice_Colony_v3/standalone/panic_room_hold.dmm:1]: bilde
bilde
Did you mean to put Xeno blood or Pred blood? Also why's the APC on the outside?

Other than that, seems pretty good/alright 👍

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: code/modules/gear_presets/survivors/shivas_snowball/panic_room_insert_shivas.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply code/modules/gear_presets/survivors/shivas_snowball/panic_room_insert_shivas.dm]: Removed.

File: maps/map_files/Ice_Colony_v3/standalone/panic_room_hold.dmm
Comment (BadAtThisGame302) [reply maps/map_files/Ice_Colony_v3/standalone/panic_room_hold.dmm:1]: Removed the pred blood, APC's for the outside Bar area.

Comment (Nanu308) [inline maps/map_files/Ice_Colony_v3/standalone/panic_room_hold.dmm:1]: mapping lgtm

**Review by Nanu308 — APPROVED** [review]

**Pull request merged**